spec/rspec/mocks/test_double_spec.rb

```diff
@@ -48,6 +48,15 @@ module Mocks
           end
         end
       end
+
+      pending "register itself with ObjectInspector" do
+        RSpec::Support::ObjectInspector.register TestDouble do |dbl|
+          "Double (#{dbl.instance_variable_get('@name')})"
+        end
```

Comment:
This needs to be removed, it's an implementation detail and doesn't belong in the spec.